cli/azd/cmd/account.go

```diff
@@ -0,0 +1,229 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/azure/azure-dev/cli/azd/internal"
+	"github.com/azure/azure-dev/cli/azd/pkg/account"
+	"github.com/azure/azure-dev/cli/azd/pkg/commands"
+	"github.com/azure/azure-dev/cli/azd/pkg/environment/azdcontext"
+	"github.com/azure/azure-dev/cli/azd/pkg/input"
+	"github.com/azure/azure-dev/cli/azd/pkg/output"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// Setup account command category
+func accountCmd(rootOptions *internal.GlobalCommandOptions) *cobra.Command {
+	root := &cobra.Command{
+		Use:   "account",
```

Comment:
I wonder if the noun here being `account` can be potentially confusing.
`azd account list` <-- Lists subscriptions, not accounts
`azd account list-locations` <-- Lists locations, not accounts

If I understood correctly, `az` provided us with a smart selection of defaults so that when you select the setting from a prompt, it defaults to those settings. I wonder if adding a final prompt on "Would you like to save subscription X and location Y as your default? ([y]es/[s]ubscription only/[l]ocation only/[n]o)?" is potentially an alternative solution.

Reply:
I purposely used the same noun/verbs that is used in the  `az cli` and also updated the help description to match what is used in `az`.

User can use the `azd account set` command to set the defaults.  We could also ask this somewhere else but going to keep it out for the time being.